index.js

```diff
@@ -77,11 +77,13 @@ module.exports.unbind = function (element, fn) {
     if (attachEvent) {
       element.detachEvent('onresize', resizeListener)
     } else if (element.__resizeTrigger__) {
-      element.__resizeTrigger__.contentDocument.defaultView.removeEventListener(
-        'resize',
-        resizeListener
-      )
-      delete element.__resizeTrigger__.contentDocument.defaultView.__resizeTrigger__
+      if (element.__resizeTrigger__.contentDocument && element.__resizeTrigger__.contentDocument.defaultView) {
```

Comment:
Please, move element.__resizeTrigger__.contentDocument to variable for better readability, and i will merge it.

Reply:
```
 var contentDocument = element.__resizeTrigger__.contentDocument;
 var defaultView = contentDocument && contentDocument.defaultView;
 if (defaultView) {
   defaultView.removeEventListener('resize', resizeListener);
   delete defaultView.__resizeTrigger__;
 }
```